website/docs/admin_hosts_houdini.md

```diff
@@ -25,12 +57,61 @@ The Shelf Set Path is used to load a .shelf file to generate your shelf set. If
 11. **⇅:** change the **shelf** order into the **shelf set**.
 12.  **⇅:** change the **tool** order in it **shelf**.
 
-# Creator plugins
+### Shelves Example
+
+Using template keys is supported but formatting keys capitalization variants is not, e.g. `{Asset}` and `{ASSET}` won't work.
+
+![Custom menu definition](assets/houdini/admin/houdini-shelf-example.png)
+
+## Creator plugins
 Enable or disable the plugins. Some of them have extra options such as defining the default subsets names.
 
-**Plugins list:** Create Arnold Ass, Create Alembic Camera, Create Composite (Image Sequence), Create Point Cache, Create Redshift ROP, Create Remote Publish, Create VDB Cache, Create USD, Create USD Model, Create USD Shading Workspace, Create USD Render.
+**Plugins list:** Create Alembic Camera, Create Arnold Ass, Create Arnold ROP, Create Composite (Image Sequence), Create Houdini Digital Asset, Create Karma ROP, Create Mantra ROP, Create PointCache (Abc), Create PointCache (Bgeo), Create Redshift Proxy, Create Redshift ROP, Create Review, Create Static Mesh, Create USD (experimental), Create USD render (experimental), Create VDB Cache, Create VRay ROP.
 
-# Publish plugings
+## Publish plugings
 Enable or disable the plugins executed at publishing.
 
-**Publish plugins list:** Validate Workfile Paths, ValidateContainers.
\ No newline at end of file
+**Publish plugins list:** Validate Latest Containers, Validate Mesh is Static, Validate Review Colorspace, Validate Subset Name, Validate Unreal Static Mesh Name,Validate workfile paths settings.
+
+---
+
+## FAQ
+
+### When will we need to modify Houdini addon code ?
+We've spent a lot of time making it possible to customize Houdini addon from settings.
+e.g. you can add your studio's menus, tools and digital assets from settings. 
+
+Still, you may need to modify Ayon addons only when :  
+- Adding/Modifying/Customizing Launch Hooks
+- Registering new Callbacks
+- Adding/Modifying/Customizing plugins (create, publish, load, Actions)
+- Adding a new Ayon launcher tray tool
+
+### How to add my menus, tools and digital assets to Houdini ?
+Mostly, you'd only need to update Environment to share these tools with your team. 
+e.g., 
+- `PYTHONPATH` to add a custom python library path
+- `HOUDINI_MENU_PATH` to add a custom menu path
+- `HOUDINI_OTL_PATH` to add a custom digital assets path
+
+Ayon allows you to add environment variables as:
+1. Global environment variable (it wil be available for all DCCs)
+2. Application specific (it wil be available for a particular DCC)
+3. Application variant specific (it wil be available for a particular DCC version)
+4. Tool (it wil be available on demand per project per asset from project Editor)
+
+For a detailed guide visit: [Ayon/Openpype Env Vars and Tools Configuration Explained](https://community.ynput.io/t/openpype-env-vars-and-tools-configuration-explained/540)
```

Comment:
Do we recommend linking to community forum posts or is the end goal to have the crucial information from topics like those to be covered in the docs itself?

Reply:
The end goal is not discussed yet.
however, I did so as I believe FAQ should have simple quick answers and the full details should live some where else. 
I used the community link because it's updated and already contains the full answer.

Reply:
I'd say it's a perfect step for now if there are no better resources - but Community topics can grow very large still making it hard to find the right info over time.